packages/pn-personafisica-webapp/src/pages/SercqSendActivation.page.tsx

```diff
@@ -0,0 +1,76 @@
+import React, { useCallback, useEffect, useState } from 'react';
+import { useTranslation } from 'react-i18next';
+import { useNavigate } from 'react-router-dom';
+
+import { Box, Typography } from '@mui/material';
+import { PnWizard, PnWizardStep } from '@pagopa-pn/pn-commons';
+import { ButtonNaked } from '@pagopa/mui-italia';
+
+import SercqSendContactWizard from '../components/Contacts/SercqSendContactWizard';
+import LoadingPageWrapper from '../components/LoadingPageWrapper/LoadingPageWrapper';
+import { getDigitalAddresses } from '../redux/contact/actions';
+import { resetState } from '../redux/contact/reducers';
+import { useAppDispatch } from '../redux/hooks';
+
+const SercqSendActivation: React.FC = () => {
```

Comment:
I should rename this into DigitalContactActivation because here we can activate the Sercq Send and the PEC. Rename also the file and the test